pkg/lib/env.go

```diff
@@ -97,6 +98,10 @@ var IsWCPVMImageRegistryEnabled = func() bool {
 	return os.Getenv(VMImageRegistryFSS) == trueString
 }
 
+var IsVMClassMigrateNamespaceFSSEnabled = func() bool {
```

Comment:
Can we change the name to be consistent with the FSS name?

Reply:
Right. Thx for catching that.